cli/build/build_test.go

```diff
@@ -24,17 +25,26 @@ func TestFillCtx(t *testing.T) {
 	require.NoError(t, os.Chdir(workDir))
 	defer os.Chdir(wd)
 	var buildCtx BuildCtx
+
+	osType, _ := util.GetOs()
+	switch osType {
+	case util.OsMacos:
+		workDir = filepath.Join("/private", workDir)
```

Comment:
There is a lot of test, which uses TempDir calls. Please, add the comment why this trick is required for MacOS.

Reply:
Added comment